src/utilities/__test__/DateUtilities.test.ts

```diff
@@ -7,9 +11,9 @@ describe("DateUtilities", () => {
   const tomorrow = now.add(1, "day").toDate();
   const today = now.toDate();
 
-  it("ToLocalDate should return date in DD/MM/YYYY format", () => {
-    expect(DateUtilities.ToLocalDate(new Date("2020-04-20"))).toEqual(
-      "20/04/2020"
+  it("ToLocalDateTime should return date in DD/MM/YYYY HH:mm format", () => {
+    expect(DateUtilities.ToLocalDateTime(new Date("2020-04-20 00:00"))).toEqual(
```

Comment:
Include a non-`00:00` time to ensure that we're not just defaulting to midnight?